pkg/cmd/roachtest/tests/decommission.go

```diff
@@ -343,7 +343,11 @@ func runDecommission(
 				return err
 			}
 			startOpts := option.DefaultStartOpts()
-			startOpts.RoachprodOpts.ExtraArgs = append(startOpts.RoachprodOpts.ExtraArgs, fmt.Sprintf("--join %s --attrs=node%d", internalAddrs[0], node))
+			extraArgs := []string{
+				fmt.Sprintf("--join %s", internalAddrs[0]),
```

Comment:
You'll need to add a `=` here or otherwise make them two strings in the slice, right?

Reply:
Woops! Fixed.